Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -309,10 +309,18 @@ static void HandleQuery()
 {
   DEBUG_LOG_FMT(GDB_STUB, "gdb: query '{}'", CommandBufferAsString() + 1);
 
-  if (!strcmp((const char*)(s_cmd_bfr + 1), "TStatus"))
-  {
-    return SendReply("T0");
-  }
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "Attached"))
+    return SendReply("1");
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "C"))
+    return SendReply("QC1");
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "fThreadInfo"))
+    return SendReply("m1");
+  else if (!strcmp((const char*)(s_cmd_bfr + 1), "sThreadInfo"))
+    return SendReply("l");
+  else if (!strncmp((const char*)(s_cmd_bfr + 1), "ThreadExtraInfo", 15))
+    return SendReply("00");
```

Comment:
This looks correct per [the documentation](https://sourceware.org/gdb/onlinedocs/gdb/General-Query-Packets.html#index-qfThreadInfo-packet), but Dolphin does actually have a way of viewing the thread list for a running game (see view &rarr; threads).  I think it'd be useful to include this there (with [`qThreadExtraInfo`](https://sourceware.org/gdb/onlinedocs/gdb/General-Query-Packets.html#index-qThreadExtraInfo-packet) showing the info in the table), though that doesn't need to happen in this PR.  Perhaps if that's done, the thread's address would be used as its thread ID, but I don't really know how threads are implemented.

Reply:
Thanks for telling me, it won't happen in this pr because to put it simply: it's not that simple, there's a WHOLE HOST of complications I have to figure out if we start to support multithreading in gdb